src/Microsoft.Health.Encryption/Customer/Health/IKeyTestProvider.cs

```diff
@@ -0,0 +1,14 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Encryption.Customer.Health;
+
+internal interface IKeyTestProvider
+{
+    Task PerformTestAsync(CancellationToken cancellationToken = default);
```

Comment:
Maybe something like `AssertHealthAsync` as "Asserts" typically throw exceptions when they fail, like this method